src/tests/integration-test

```diff
@@ -246,7 +251,8 @@ class UDisksTestCase(unittest.TestCase):
         If cd is True, return the CD device, otherwise the hard disk device.
         """
         block = klass.udisks_block(partition, cd)
-        return klass.client.get_object(block.get_object_path()).get_filesystem()
+        return klass.client.get_object(
+            block.get_object_path()).get_filesystem()
```

Comment:
> As for tests being as good as the code -- sure, let's start with the code then, it's much worse than our tests.

From a code quality standpoint I agree, but from a formatting point I disagree.  Most of my fixes to PR's are to address white space issues with my **C** code changes.  This seems like a stark contrast to the python test code where anything is virtually allowed.  That's why I like tools to enforce formatting.  Then you can just say run this and fix what ever spits out, saves time for everyone.